components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/utils/APIImportUtil.java

```diff
@@ -434,6 +434,7 @@ public static void importAPI(String pathToArchive, String currentUser, boolean i
             // Change API lifecycle if state transition is required
             if (StringUtils.isNotEmpty(lifecycleAction)) {
                 log.info("Changing lifecycle from " + currentStatus + " to " + targetStatus);
+                apiProvider.changeAPILCCheckListItems(importedApi.getId(), 1, true);
```

Comment:
what is that magic number 1 ? :P

Reply:
It was used to refer the first attribute  of the "CheckListItems"

Reply:
Done with 1291d974f0ea49f2eb61114428f473fb3811d844